runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingSpotOverlay.java

```diff
@@ -124,6 +125,13 @@ public Dimension render(Graphics2D graphics)
 			if (config.showSpotTiles())
 			{
 				Polygon poly = npc.getCanvasTilePoly();
+
+				if (spot == FishingSpot.COMMON_TENCH
+					&& npc.getWorldLocation().distanceTo2D(client.getLocalPlayer().getWorldLocation()) <= 3)
```

Comment:
This 3 should be defined somewhere as a constant since it is used multiple places.